BoostTestAdapter/BoostTestExecutor.cs

```diff
@@ -346,20 +350,20 @@ private void RunBoostTests(IEnumerable<TestRun> testBatches, IRunContext runCont
 
                 try
                 {
-                    Logger.Info("{0}:   -> [{1}]", ((runContext.IsBeingDebugged) ? "Debugging" : "Executing"), string.Join(", ", batch.Tests));
+                    Logger.Info(((runContext.IsBeingDebugged) ? Resources.Debugging : Resources.Executing), string.Join(", ", batch.Tests));
 
                     using (TemporaryFile report = new TemporaryFile(batch.Arguments.ReportFile))
                     using (TemporaryFile log    = new TemporaryFile(batch.Arguments.LogFile))
                     using (TemporaryFile stdout = new TemporaryFile(batch.Arguments.StandardOutFile))
                     using (TemporaryFile stderr = new TemporaryFile(batch.Arguments.StandardErrorFile))
                     {
-                        Logger.Debug("Working directory: {0}", batch.Arguments.WorkingDirectory ?? "(null)");
-                        Logger.Debug("Report file      : {0}", batch.Arguments.ReportFile);
-                        Logger.Debug("Log file         : {0}", batch.Arguments.LogFile);
-                        Logger.Debug("StdOut file      : {0}", batch.Arguments.StandardOutFile ?? "(null)");
-                        Logger.Debug("StdErr file      : {0}", batch.Arguments.StandardErrorFile ?? "(null)");
+                        Logger.Debug(Resources.WorkingDirectory, batch.Arguments.WorkingDirectory ?? "(null)");

```

Comment:
Resx comment on these strings should indicate that we want to have them aligned to each other.

Reply:
Resolved